MockKit/MockService.swift

```diff
@@ -151,3 +155,29 @@ extension MockService: RemoteDataService {
     }
     
 }
+
+extension MockService: VersionCheckService {
+    public func checkVersion(bundleIdentifier: String, currentVersion: String, completion: @escaping (Result<VersionUpdate, Error>) -> Void) {
+        record("[VersionCheckService] Version checked \(currentVersion)")
+        completion(.success(versionUpdate))
+    }
+}
+
+extension VersionUpdate: RawRepresentable {
```

Comment:
Why not just make `VersionUpdate` a `String enum` and then get this for free?

Reply:
I do that would mess up the Comparable bit, but I don't see why that is needed. I must be missing something.

Reply:
I need `Comparable` as you saw in the other PR.